frontend/src/modules/transactions/components/ExchangeDetail.tsx

```diff
@@ -0,0 +1,218 @@
+                </div>
+              </div>
+
+              <div className="d-flex mt-2">
+                <div className="product-image">
+                  <img
+                    src={currentProduct?.imageUrl}
+                    alt="product"
+                    width={100}
+                    height={100}
+                  />
+                </div>
+                <div className="flex-column ms-3 gap-1">
+                  <div className="title bold-25 text-color-quaternary">
+                    {currentProduct?.title}
+                  </div>
+                  <div className="owner-full-name regular-14 text-color-quaternary text-color-tertiary">
+                    {`Được đăng bởi ${currentProduct?.owner?.firstName} ${currentProduct?.owner?.lastName}`}
+                  </div>
+                  <div className="price semibold-20 text-color-tertiary">
+                    {formatToVietnameseCurrency(currentProduct?.suggestedPrice)}
+                  </div>
+                </div>
+              </div>
+            </div>
+            {/* Thông tin người mua */}
+            <div className="col-6 d-flex flex-column gap-3 halfright-box">
+              <div className="owner-full-name label bold-25 text-color-quaternary">
+                {`Bạn`}
+              </div>
+              <div className="phone-number d-flex gap-1">
+                <div className="label semibold-14 text-color-quaternary">
+                  {'Số điện thoại:'}
+                </div>
+                <div className="value regular-14 text-color-tertiary">
+                  0987 456 789
+                </div>
+              </div>
+              <div className="address d-flex gap-1">
+                <div className="label semibold-14 text-color-quaternary">
+                  {'Địa chỉ:'}
+                </div>
+                <div className="value regular-14 text-color-tertiary">
+                  Phường 13, Quận Bình Thạnh, Tp Hồ Chí Minh
+                </div>
+              </div>
+              <div className="d-flex mt-2">
+                <div className="product-image">
+                  <img
+                    src={myProducts?.imageUrl}
+                    alt="product"
+                    width={100}
+                    height={100}
+                  />
+                </div>
+                <div className="flex-column ms-3 gap-1">
+                  <div className="title bold-25 text-color-quaternary">
+                    {myProducts?.title}
+                  </div>
+                  <div className="owner-full-name regular-14 text-color-quaternary text-color-tertiary">
+                    {`Được đăng bởi bạn`}
+                  </div>
+                  <div className="price semibold-20 text-color-tertiary">
+                    {formatToVietnameseCurrency(myProducts?.suggestedPrice)}
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          {/* Stepper */}
```

Comment:
Never comment code, if this is not done, create TODO or JIRA ticket implement later